src/workloads/c2c/CollectionCopy.yml

```diff
@@ -0,0 +1,75 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/product-perf"
+Description: |
+  This workload is a short version load used to test Mongosync Collection Copy stage performance.
+  The workload starts Mongosync on a cluster with an preloaded initial dataset.
+
+Keywords:
+- c2c
+- replication
+- collection copy
+- cluster to cluster sync
+
+# These two values should match those are the top of MixPhases.yml
+dbname: &dbname mongosync_test
+DocumentCount: &NumDocs 100000
+CollectionCount: &NumColls 1
+ScriptsPath: &scriptsPath ./MongosyncScripts.yml
+
+Parameters:
+- &Nop {Nop: true}
+
+Clients:
+  Default:
+    QueryOptions:
+      maxPoolSize: 500
+
+Actors:
+- Name: Setup
+  Type: Loader
+  Threads: 1
+  Phases:
+  - Repeat: 1
+    BatchSize: 100
+    Threads: 1
+    DocumentCount: *NumDocs
+    Database: *dbname
+    CollectionCount: *NumColls
+    Document: &doc
```

Comment:
If this is the same document structure as the CEA workload, what are your thoughts on using `LoadConfig` here to re-use some of the code?

Reply:
Good idea!

Reply:
I moved the command insertion phase to `MongosyncScripts.yml`